containers/java/clipper-java-container/src/main/java/clipper/container/app/ModelContainer.java

```diff
@@ -6,17 +6,18 @@
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.nio.IntBuffer;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
 
 import clipper.container.app.data.*;
 import org.zeromq.ZMQ;
 
 class ModelContainer<I extends DataVector<?>> {
-  private static String CONNECTION_ADDRESS = "tcp://%s:%s";
+  private static final String CONNECTION_ADDRESS = "tcp://%s:%s";
+  private static final long SOCKET_POLLING_TIMEOUT_MILLIS = 5000;
+  private static final long SOCKET_ACTIVITY_TIMEOUT_MILLIS = 60000;
```

Comment:
Runtime configuration of these parameters will create behavioral inconsistencies w.r.t RPC across container instances. I think all containers corresponding to a given language should maintain a consistent RPC protocol.